src/autoDetector.js

```
@@ -0,0 +1,59 @@
+  let col = 0;
+  for (let i = 0; i < letter.length; i++) {
+    col = col * 26 + (letter.charCodeAt(i) - 64);
+  }
+  return col - 1; // zero-based
+}
+
+function parseCellAddress(addr) {
+  const match = /^([A-Z]+)(\d+)$/i.exec(addr);
+  if (!match) return null;
+  const colLetters = match[1].toUpperCase();
+  const rowNumber = parseInt(match[2], 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)